research/src/pages/light-dismiss/light-dismiss.research.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Checkbox Concepts
+path: /components/light-dismiss.research
```

Comment:
I'm curious if this should sit on its own right now? Maybe it should be a part of `<popup>`?

Reply:
My idea is that it's an independant concept that applies to `<popup>` but can imagine it "loops back" at some point.